.github/workflows/ci.yaml

```diff
@@ -35,7 +35,7 @@ jobs:
         juju:
           - agent: 2.9.50  # renovate: juju-agent-pin-minor
             libjuju: ==2.9.49.0  # renovate: latest libjuju 2
-          - agent: 3.4.5  # renovate: juju-agent-pin-minor
+          - agent: 3.5.3  # renovate: juju-agent-pin-minor
```

Comment:
Test are light here, if we want we can run both versions.

Reply:
3/stable is enough here, IMHO, it is an internal test app and not a production charm.